tests/pipelines/test_pipelines_document_question_answering.py

```diff
@@ -82,7 +82,8 @@ def get_test_pipeline(self, model, tokenizer, processor):
                 "word_boxes": word_boxes,
             },
             {
-                "image": None,
+                # if image processor is available, we always need to prepare `image`, otherwise let's test without it
+                "image": load_image(image) if processor is not None else None,
```

Comment:
I don't understand this.

Let's just remove this test case, `image:None ` in a document-qa pipeline doesn't  make a whole lot of sense in general imo.

Reply:
OK! Just want to pointing out that when this pipeline test is run with `LayoutLM` model (not `v2`), it works fine, as it seems there is no image processor associated in this case.

**That's why I am not sure if this document-qa pipeline should or not (or both)  contain the image inputs.**

Reply:
> That's why I am not sure if this document-qa pipeline should or not (or both) contain the image inputs.

I think support was added because of LayoutLM. Strange (and sad) that V2 and V3 were never really tested.

Personally I would simply remove the `None` from the test, it's too niche to be valuable, imo. (What kind of document question answering, answers without a document ! :D That's called plain question-answering :) )

Reply:
OK